docs/testing/testcafe_functional_testing.md

```diff
@@ -0,0 +1,150 @@
+---
+id: testcafe_functional_testing
+title: Functional Testing with TestCafe
+sidebar_label: TestCafe - Functional Testing
+---
+
+## TestCafe: Functional tests
+
+Templated sample TestCafe automation framework for PROJECT_NAME, bootstrapped using `@amidostacks/scaffolding-cli`.
+
+## Getting started
+
+Ensure that [Node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/) are installed with the versions specified in [package.json](./package.json).
+
+### Dependencies
+
+```bash
+npm install
+```
+
+### End to End (E2E) Tests
+
+```bash
+npm run test:e2e
+```
+
+This should launch [TestCafe](https://devexpress.github.io/testcafe/documentation/getting-started/) and run the tests in the browsers specified in [.testcaferc.json](./.testcaferc.json) in the headless mode.
+
+### Accessibility Tests with aXe
+
```

Comment:
This requires some explanation as you have gone from Functional Testing with TestCafe to Accessibility testing with aXe.  A section renamed as Adding accessibility Tests with aXe to TestCafe perhaps.  You should also move this section until after you have fully gone through functional testing, so further down the page